migration-engine/connectors/sql-migration-connector/src/sql_schema_calculator/sql_schema_calculator_flavour/sqlite.rs

```diff
@@ -9,7 +10,11 @@ impl SqlSchemaCalculatorFlavour for SqliteFlavour {
     }
 
     // Integer primary keys on SQLite are automatically assigned the rowid, which means they are automatically autoincrementing.
-    fn field_is_implicit_autoincrement_primary_key(&self, field: &ScalarFieldWalker<'_>) -> bool {
-        field.is_id() && field.field_type().is_int()
+    fn field_is_implicit_autoincrement_primary_key(
+        &self,
+        model: &ModelWalker<'_>,
+        field: &ScalarFieldWalker<'_>,
```

Comment:
You should be able to do `field.model()` instead of passing the model here.

Reply:
Done.